JSDemos/Demos/TreeList/BatchEditing/description.md

```diff
@@ -1 +1,7 @@
-Batch editing is&nbsp;very similar to&nbsp;cell editing in&nbsp;that a&nbsp;user edits data cell by&nbsp;cell. The only difference is&nbsp;that with batch editing a&nbsp;user needs to&nbsp;click the _&laquo;Save changes&raquo;_ button to&nbsp;actually save changes. Only then the TreeList will collect all changes in&nbsp;a&nbsp;batch and send them to&nbsp;the data source.
\ No newline at end of file
+You can use batch edit mode to defer saving multiple changes. In this mode, users edit data one cell at a time. The changes are stored in a buffer and can be discarded before the **Save** button is clicked.
+
+To enable batch edit mode, set the [mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) property to *"batch"* and enable the [editing](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#mode) object's [allowUpdating](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowUpdating), [allowAdding](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowAdding), and [allowDeleting](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/editing/#allowDeleting) properties to allow edit operations.
+
+This demo uses the [onInitNewRow](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onInitNewRow) handler to populate cells of a new row with default values. This function sets Not Started and as the initial task status and specifies the current date as the initial start and due dates.
```

Comment:
BTW, I do not understand why the developers decided to show this function i this demo. Anyway:

```suggestion
This demo uses the [onInitNewRow](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/#onInitNewRow) handler to populate cells of a new row with default values. This function sets `Not Started` as the task status and specifies the current date as the start and due dates.
```